scripts/patch-realsense-ubuntu-odroid-xu4-4.14.sh

```diff
@@ -0,0 +1,191 @@
+echo "Kernel branch: " $kernel_branch
+# Construct branch name from distribution codename {xenial,bionic,..} and kernel version
+#ubuntu_codename=`. /etc/os-release; echo ${UBUNTU_CODENAME/*, /}`
+#if [ -z "$UBUNTU_CODENAME" ];
+#then
+	# Trusty Tahr shall use xenial code base
+#	ubuntu_codename="xenial"
+#	retpoline_retrofit=1
+#fi
+ubuntu_codename="bionic"
+
+#kernel_name="ubuntu-${ubuntu_codename}-$kernel_branch"
+kernel_name="odroid_bionic"
+
+#Distribution-specific packages
+if [ ${ubuntu_codename} == "bionic" ];
+then
+	require_package libelf-dev
+	require_package elfutils
+fi
+
+
+# Get the linux kernel and change into source tree
+#[ ! -d ${kernel_name} ] && git clone -b $kernel_branch git://#kernel.ubuntu.com/ubuntu/ubuntu-${ubuntu_codename}.git --depth #1 ./${kernel_name}
+[ ! -d ${kernel_name} ] && git clone --depth 1 https://github.com/hardkernel/linux -b odroidxu4-4.14.y ./${kernel_name}
+cd ${kernel_name}
+
+# Verify that there are no trailing changes., warn the user to make corrective action if needed
+if [ $(git status | grep 'modified:' | wc -l) -ne 0 ];
+then
+	echo -e "\e[36mThe kernel has modified files:\e[0m"
+	git status | grep 'modified:'
+	echo -e "\e[36mProceeding will reset all local kernel changes. Press 'n' within 10 seconds to abort the operation"
+	set +e
+	read -n 1 -t 10 -r -p "Do you want to proceed? [Y/n]" response
+	set -e
+	response=${response,,}    # tolower
+	if [[ $response =~ ^(n|N)$ ]]; 
+	then
+		echo -e "\e[41mScript has been aborted on user requiest. Please resolve the modified files are rerun\e[0m"
+		exit 1
+	else
+		echo -e "\e[0m"
+		echo -e "\e[32mUpdate the folder content with the latest from mainline branch\e[0m"
+		#git fetch origin $kernel_branch --depth 1
+		git fetch origin odroidxu4-4.14.y --depth 1
+		printf "Resetting local changes in %s folder\n " ${kernel_name}
+		#git reset --hard $kernel_branch
+        git reset --hard odroidxu4-4.14.y
+	fi
+fi
+
+#Check if we need to apply patches or get reload stock drivers (Developers' option)
+[ "$#" -ne 0 -a "$1" == "reset" ] && reset_driver=1 || reset_driver=0
+
+if [ $reset_driver -eq 1 ];
+then 
+	echo -e "\e[43mUser requested to rebuild and reinstall ubuntu-${ubuntu_codename} stock drivers\e[0m"
+else
+	# Patching kernel for RealSense devices
+	echo -e "\e[32mApplying realsense-uvc patch\e[0m"
+	patch -p1 < ../scripts/realsense-camera-formats_ubuntu-${ubuntu_codename}-${kernel_branch}.patch
+	echo -e "\e[32mApplying realsense-metadata patch\e[0m"
+	patch -p1 < ../scripts/realsense-metadata-ubuntu-${ubuntu_codename}-${kernel_branch}.patch
+	echo -e "\e[32mApplying realsense-hid patch\e[0m"
+	patch -p1 < ../scripts/realsense-hid-ubuntu-${ubuntu_codename}-${kernel_branch}.patch
+	echo -e "\e[32mApplying realsense-powerlinefrequency-fix patch\e[0m"
+	patch -p1 < ../scripts/realsense-powerlinefrequency-control-fix.patch
+fi
+
+# Copy configuration
+sudo cp /usr/src/linux-$(uname -r)/.config .
+sudo cp /usr/src/linux-$(uname -r)/Module.symvers .
+
```

Comment:
this path does not exist. propose following change


```
# Copy configuration
sudo cp /lib/modules/$(uname -r)/build/.config .
sudo cp /lib/modules/$(uname -r)/build/Module.symvers .
```

Reply:
I second this. I used the discussion referenced, and updated the lines to indicate the correct path of /lib/modules/. Works like a charm.